sql/core/src/test/resources/sql-tests/results/pgSQL/window.sql.out

```diff
@@ -0,0 +1,2554 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+
+
+-- !query 115
+DROP TABLE empsalary
+-- !query 115 schema
+struct<>
+-- !query 115 output
+
+
+
+-- !query 116
+SELECT i,AVG(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v)
+-- !query 116 schema
+struct<i:int,avg(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):double>
+-- !query 116 output
+1	1.5
+2	2.0
+3	NULL
+4	NULL
+
+
+-- !query 117
+SELECT i,AVG(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v)
+-- !query 117 schema
+struct<i:int,avg(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):double>
+-- !query 117 output
+1	1.5
+2	2.0
+3	NULL
+4	NULL
+
+
+-- !query 118
+SELECT i,AVG(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v)
+-- !query 118 schema
+struct<i:int,avg(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):double>
+-- !query 118 output
+1	1.5
+2	2.0
+3	NULL
+4	NULL
+
+
+-- !query 119
+SELECT i,AVG(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1.5),(2,2.5),(3,NULL),(4,NULL)) t(i,v)
+-- !query 119 schema
+struct<i:int,avg(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):decimal(6,5)>
+-- !query 119 output
+1	2
+2	2.5
+3	NULL
+4	NULL
+
+
+-- !query 120
+SELECT i,AVG(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,'1 sec'),(2,'2 sec'),(3,NULL),(4,NULL)) t(i,v)
+-- !query 120 schema
+struct<i:int,avg(CAST(v AS DOUBLE)) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):double>
+-- !query 120 output
+1	NULL
+2	NULL
+3	NULL
+4	NULL
+
+
+-- !query 121
+SELECT i,SUM(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v)
+-- !query 121 schema
+struct<i:int,sum(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):bigint>
+-- !query 121 output
+1	3
+2	2
+3	NULL
+4	NULL
+
+
+-- !query 122
+SELECT i,SUM(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v)
+-- !query 122 schema
+struct<i:int,sum(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):bigint>
+-- !query 122 output
+1	3
+2	2
+3	NULL
+4	NULL
+
+
+-- !query 123
+SELECT i,SUM(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
+  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v)
+-- !query 123 schema
+struct<i:int,sum(v) OVER (ORDER BY i ASC NULLS FIRST ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING):bigint>
+-- !query 123 output
+1	3
+2	2
+3	NULL
+4	NULL
+
+
+-- !query 124
+SELECT i,SUM(v) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
```

Comment:
money test? https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/expected/window.out#L3421

Reply:
Hmm I think that casting with `::` is actually present in pgsql for legacy reasons (i.e: they will probably remove it at some point). Do you think that I should cast it in Spark using another method or that I should JIRA the fact that Spark didn't recognize that it is actually a column of type `money`?

Reply:
IMO its ok just to comment out the original query and then describe why its commented out.

Reply:
Great. I added the following comment: `The cast syntax is present in PgSQL for legacy reasons and Spark will not recognize a money field`. Do you think that it is ok? It makes sense to JIRA the missing money type?

Reply:
I think we don't need to.